styles/style.css

```diff
@@ -0,0 +1,210 @@
+/* * {
+  border: black 0.5px solid;
+} */
+
+body {
+  font-family: 'Roboto', sans-serif;
+  background-color: seashell;
+}
+
+.menu {
+  display: grid;
+  grid-template-columns: 3fr 1fr 1fr;
+  margin: auto;
+  padding: 10px 10px 10px 10px;
+  list-style-type: none;
+  position: fixed;
+  top: 0;
+  width: 100%;
+
+}
+
+h1 {
+  font-size: 4em;
+  font-family: 'Muli', sans-serif;
+}
+
+.welcome {
+  display: flex;
+  flex-flow: column wrap;
+  align-content: space-between;
+  text-align: center;
+  font-family: Arial, bold, sans-serif;
+  margin: 180px 0px 0px 0px;
+  padding: 5px 0px 15em 0px;
+}
+
+.welcome sub {
+  font-size: 1.2em;
+  font-family: 'Noto Serif JP', sans-serif;
+  font-weight: bold;
+  text-transform: capitalize;
+  font-stretch: ultra-expanded;
+  margin-bottom: 2em;
+}
+
+.welcome a[href] {
+  font-size: .9rem;
+  color: gray;
+  margin: auto;
+  border: 2px darkgray solid;
+  border-radius: 10px;
+  padding: 5px;
+  text-decoration: none;
+  max-width: 10em;
+  background-color: white;
+}
+
+.quote {
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  align-content: space-evenly;
+  justify-content: center;
+  font-style: italic;
+  font-size: 20px;
+  font-family: 'Times New Roman', cursive;
+  margin: 0px 0px 100px 0px;
+  line-height: 2em;
+  padding: 6em 5px 5em 5px;
+}
+
+.quote blockquote::before, blockquote::after {
+  content: "\"";
+}
+
+.quote blockquote {
+  padding: 1em 2em 1em 2em;
+}
+
+.quote p {
+  align-self: flex-end;
+  margin-right: 15%;
+}
+
+.quote p::before {
+  content: "--"
+}
+
+.why {
+  display: grid;
+  grid-template-rows: 1fr 3fr;
+  margin-top: -1em;
+}
+
+h2 {
+  text-align: center;
+  font-family: Arial, bold, sans-serif;
+}
+
+.answers {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-column-gap: 20px;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+.answerbox img {
+  grid-area: 1/1/3/2;
+  float: left;
+  margin-right: 0.5em;
+}
+
+.answerbox p {
+  padding: 1em 1em;
+  margin-top: -0.5em;
+}
+
+.team {
```

Comment:
this is the one part of your site that isn't exactly "responsive" (as a i understand it) -- not that this was a project requirement or anything -- BUT i'm under the impression that minmax is supposed to help with this. not sure if creating extra rows or columns would allow the images to automatically flow there if needed? idk things.